Modules/@babylonjs/react-native/package.json

```diff
@@ -29,9 +29,7 @@
   },
   "peerDependencies": {
     "@babylonjs/core": "^5.0.0-rc.4",
-    "react": ">=16.13.1",
-    "react-native": ">=0.63.1",
-    "react-native-permissions": "^2.1.4"
```

Comment:
Hmm so you were thinking move these dependencies to the binary packages, and I was thinking have these only in this package. I guess it's weird because we have both TypeScript and native dependencies on these packages. I think I feel like this is a lower level package that does in fact have these dependencies, so I still think it probably makes more sense to have them here, but if you have a different perspective I'm definitely open.

Reply:
Makes sense. Fixed.